chat-server/pkg/chat_v1/chat.pb.go

```diff
@@ -0,0 +1,480 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
This file should not be here. It's supposed that it will be generated by other developer using `protoc`